src/equations/compressible_euler_1d.jl

```diff
@@ -30,7 +30,7 @@ end
 
 A smooth initial condition used for convergence tests in combination with
 [`source_terms_convergence_test`](@ref)
-(and [`boundary_condition_convergence_test`](@ref) in non-periodic domains).
+(and [`BoundaryConditionDirichlet(initial_condition_convergence_test)`](@ref) in non-periodic domains).
```

Comment:
Did you check whether this link (and similar ones below) resolves as desired?

Reply:
Not yet. I'll check this later, but it didn't cause any warnings/errors in the Documentation job.

Reply:
I just checked, the whole Text `BoundaryConditionDirichlet(initial_condition_convergence_test)` links to `#Trixi.BoundaryConditionDirichlet` as expected.